components/atom/card/src/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+const BASE_CLASS = 'sui-AtomCard'
+const CLASS_LINK = `${BASE_CLASS}-link`
+const CLASS_MEDIA = `${BASE_CLASS}-media`
+const CLASS_INFO = `${BASE_CLASS}-info`
+
+const AtomCardDiv = ({
+  className, // eslint-disable-line react/prop-types
+  media: Media, // eslint-disable-line react/prop-types
+  content: Content, // eslint-disable-line react/prop-types
+  ...props
+}) => (
+  <div className={className} {...props}>
```

Comment:
Any props can be added to the DIV.
Aren't we supposed tu limit the props of the component ? 
@jcger ?

Reply:
Fixed!